### DISCUSSION TITLE
Как понять эту ошибку внутри ядра aiogram?

### DISCUSSION THREAD

**Discussion by ikattz** [aiogram/aiogram#1166] [Q&A]
`base_events.py [ LINE:1738] ERROR    [2023-04-16 20:54:05,112] Task exception was never retrieved
future: <Task finished name='Task-1652' coro=<Handler.notify() done, defined at /usr/local/lib/python3.9/dist-packages/aiogram/dispatcher/handler.py:84> exception=TypeError('on_pre_process_error() takes 4 positional arguments but 5 were given')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/aiogram/dispatcher/handler.py", line 100, in notify
    await self.dispatcher.middleware.trigger(f"pre_process_{self.middleware_key}", args + (data,))
  File "/usr/local/lib/python3.9/dist-packages/aiogram/dispatcher/middlewares.py", line 53, in trigger
    await app.trigger(action, args)
  File "/usr/local/lib/python3.9/dist-packages/aiogram/dispatcher/middlewares.py", line 106, in trigger
    await handler(*args)
TypeError: on_pre_process_error() takes 4 positional arguments but 5 were given
   logging.py [ LINE:28  ] DEBUG    [2023-04-16 20:57:54,535] Received update [ID:574978553]`

Что это значит? Ошибка внутри ядра где-то почему-то... Начала вылетать на рабочем проекте. не могу отследить причину.

**Comment by KytryshAndrii** [comment]
Вутри ядра в функцию middlewares.py подано 5 аргументов вместо нужных пяти. Дожно быть даешь больше аргументов в метод чем нужно. Попробуй документацию . Если прикрепишь код, будет проще отвловить баг 

https://docs.aiogram.dev/en/latest/examples/middleware_and_antiflood.html

The middlewares.py function in the kernel has 5 arguments instead of the necessary five. You must be giving more arguments to the method than necessary. Try the documentation. If you attach the code, it'll be easier to catch the bug.

**Reply by ikattz** [reply]
А как понять в какой метод то? Просто подозрительно, до этого работало все прекрасно. Почему-то в консоли только ошибка ядра, нет первопричины из моего кода.

**Comment by JrooTJunior** [comment]
Use english language here if you really want help.

Please provide minimmum reproducible example to check this issue.
